packages/eas-cli/src/commands/channel/view.ts

```diff
@@ -124,17 +187,42 @@ export default class ChannelView extends Command {
       return;
     }
 
+    const { branchMapping, isRollout, rolloutPercent } = getBranchMapping(
+      getUpdateChannelByNameForAppresult
+    );
+
     const table = new Table({
-      head: ['branch', 'update', 'message', 'created-at', 'actor'],
+      head: [
+        'branch',
+        ...(isRollout ? ['rollout percent'] : []),
+        'active update group',
+        'update message',
+        'last publish',
+        'actor',
+      ],
       wordWrap: true,
     });
 
-    for (const branch of channel.updateBranches) {
-      // TODO (cedric): refactor when multiple branches per channel are available
-      const update = branch.updates[0];
+    for (const index in branchMapping.data) {
+      if (parseInt(index, 10) > 1) {
+        throw new Error('Branch Mapping data must have lenght less than or equal to 2.');
```

Comment:
I'm a bit concerned about this "duplicate" validation logic, compared to the other validation in `getBranchMapping`. If this needs to ship asap, maybe add a note that this is similar to that file? Or preferably, if we have time, maybe export a validation method to validate an arbitrary object from somewhere (like utils?).

Reply:
I was thinking that the validation in `getBranchMapping` could expand with this still being restricted to just 2. 

Like what if we did more complex branch mappings, I would want the developer to realize that this output loop was only capable of handling the case of `n=2`.  In that case we may want to keep this loop as is, but write a different function for printing `n>2`.